process/project-lifecycle-documents/zarf_sandbox_stage.md

```diff
@@ -0,0 +1,34 @@
+## Application for creating a new project at Sandbox stage
+
+### List of project maintainers
+The project must have a minimum of three maintainers with a minimum of two different organizational affiliations.
+  * Sarah Christoff, Defense Unicorns, @sscristoff-du
+  * Austin Abro, Defense Unicorns, @AustinAbro321
+  * Lucas Rodriguez, Defense Unicorns, @lucasrod16
+  * Danny Gershman, Radius Method, @dgershman
+
+### Sponsor
+Most projects will report to an existing OpenSSF Working Group, although in some cases a project may report directly to the TAC. The project commits to providing quarterly updates on progress to the group they report to.
+  * WG Supply Chain Integrity
+
+### Mission of the project
+The project must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+  
+  * Zarf eliminates the complexity of air gap software delivery for Kubernetes clusters and cloud-native workloads using a declarative packaging strategy to support DevSecOps in offline and semi-connected environments.
+
+### IP policy and licensing due diligence
+When contributing an existing Project to the OpenSSF, the contribution must undergo license and IP due diligence by the Linux Foundation (LF).
+  * Not applicable - There has been no formal trademark filed by Defense Unicorns.
```

Comment:
I don't believe this is the correct answer. The requirement applies to all projects coming from outside OpenSSF.
I suggest this be changed once the due diligence is complete.